Code/Editor/EditorEngineProcessFramework/EngineProcess/EngineProcessMessages.h

```diff
@@ -223,6 +223,7 @@ class EZ_EDITORENGINEPROCESSFRAMEWORK_DLL ezViewRedrawMsgToEngine : public ezEdi
   ezUInt16 m_uiWindowHeight;
   bool m_bUpdatePickingData;
   bool m_bEnablePickingSelected;
+  bool m_bEnablePickTranslucent;
```

Comment:
Should be called Transparent. We use tranclucent for objects with subsurface scattering.

Reply:
Ok. I used translucent, because that's what UE does.